exercises/ocr-numbers/description.md

```diff
@@ -60,18 +60,18 @@ Is converted to "1234567890"
 Update your program to handle multiple numbers, one per line. When converting several lines, join the lines with commas.
 
 ```text
-    _  _ 
+    _  _
   | _| _|
   ||_  _|
-         
-    _  _ 
-|_||_ |_ 
+
+    _  _
+|_||_ |_
   | _||_|
-         
- _  _  _ 
+
+ _  _  _
   ||_||_|
   ||_| _|
-         
+
```

Comment:
The whitespace here is significant and not allowed to be removed

Reply:
What about changing it to this?

```text
    _  _  #
  | _| _| #
  ||_  _| #
          # (blank fourth row)
```

Reply:
With literal double crosses?

I think that would lead to similar discussion we had on another exercise where a border was drawn in the examples for visual clarification but not in the test suits for easier programmatic access... I think it was the minesweeper exercise.

Just let the whitespace where it belongs and don't add arbitrary characters which could be misinterpreted

Reply:
Alright. I reverted the changes to this file.